### ISSUE TITLE
qm9_nn_conv.py yields very large and inconsistent results

### ISSUE THREAD

**Issue by xOmer12** [pyg-team/pytorch_geometric#10517]
### 🐛 Describe the bug

Hi,
I am trying to train GNNs for the QM9 dataset different targets using the qm9_nn_conv.py script in the examples folder, however, the training itself is very unstable and results does not match existing papers. For example, on target 1:

<img width="701" height="166" alt="Image" src="https://github.com/user-attachments/assets/8cff84e0-9e52-4356-b379-d66bcde10048" />

after few epochs, the test MAE grows very large, and in general the test MAE and validation MAE are not close to what the model should achieve. This happens across several tasks and I could not find why. It happens on the exacts script from the github repo with the exact hyperparameters. I would be glad for any help, thanks in advanc!

### Versions

<img width="517" height="512" alt="Image" src="https://github.com/user-attachments/assets/eac9a2f5-d697-407a-8bbc-093eda4b6a49" />

**Comment by MeghanaShanthappa** [comment]
Hi, I tried to reproduce this issue with target = 1, but I could not reproduce the MAE explosion so far.

My environment:
- PyG 2.7.0
- PyTorch 2.8.0+cu128
- CUDA available: True
- GPU: RTX 4090

I ran examples/qm9_nn_conv.py after changing target = 1 and got:

Epoch 001: Val MAE 1.3761, Test MAE 1.3703
Epoch 005: Val MAE 0.8279, Test MAE 0.8322
Epoch 010: Val MAE 0.7186, Test MAE 0.6774

Could you share the exact environment, command, target, and seed needed to reproduce the very large MAE behavior?
